x11-themes/kde-plasma-chili/kde-plasma-chili-0.5.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="The hottest login theme around for KDE Plasma 5"
+HOMEPAGE="https://github.com/MarianArlt/kde-plasma-chili"
+SRC_URI="https://github.com/MarianArlt/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	kde-plasma/plasma-meta
+	x11-misc/sddm
+"
+
+DOCS=(
```

Comment:
These should all be part of default `einstalldocs` array, so no need to specify them here (please do test it though).

Reply:
Yes, except **CHANGELOG.md**.